sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -672,15 +672,6 @@ class Analyzer(
    */
   object ResolveTables extends Rule[LogicalPlan] {
     def apply(plan: LogicalPlan): LogicalPlan = plan.resolveOperatorsUp {
-      case u: UnresolvedRelation =>
-        lookupV2Relation(u.multipartIdentifier)
-          .getOrElse(u)
-
-      case i @ InsertIntoStatement(u: UnresolvedRelation, _, _, _, _) if i.query.resolved =>
-        lookupV2Relation(u.multipartIdentifier)
-          .map(v2Relation => i.copy(table = v2Relation))
-          .getOrElse(i)
-
       case u: UnresolvedV2Relation =>
         CatalogV2Util.loadTable(u.catalog, u.tableName).map { table =>
           DataSourceV2Relation.create(table)
```

Comment:
ResolveTables now only resolves UnresolvedV2Relation. Does it still make sense to have it as separate rule?

Reply:
We will add more in https://github.com/apache/spark/pull/25955